### PR TITLE
added enfageorge to list

### PR DESCRIPTION
**Pull request by beingenfa**

### CODE REVIEW

**Comment by datapythonista** [comment]
Good job @enfageorge. Do you mind in a separate PR to change the title of the page? May be you can find us a good name?

**Comment by beingenfa** [comment]
> Good job @enfageorge. Do you mind in a separate PR to change the title of the page? May be you can find us a good name?

I'll take suggestions from the team and do that as soon as we fix on one.

**Comment by sparalic** [comment]
@enfageorge—> Pandanistas

**Comment by datapythonista** [comment]
pandas girls or pydata girls are not very original, but I guess you're aware of https://djangogirls.org/ ;)

**Comment by datapythonista** [comment]
and you surely know https://www.pyladies.com/

**Comment by datapythonista** [comment]
@enfageorge we will go back to the name later, can you for now just remove the `Mentoring` in the title, and just leave `New pandas contributors (team 2019)` please? In a new PR.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ List of the [GitHub](https://github.com) usernames of all the people interested
 - [martinagvilas](https://github.com/martinagvilas)
 - [galuhsahid](https://github.com/galuhsahid)
 - [Bhavani Ravi](https://github.com/bhavaniravi)
+
```
Comment (datapythonista) [inline README.md]: This blank line is likely to divide the list in two. Can you remove it please?

Comment (beingenfa) [reply README.md]: done. Sorry for the typo.

**Review by datapythonista — APPROVED** [review]
Cool, this looks great now.

Can you work on https://github.com/python-sprints/pandas-mentoring/pull/26#issuecomment-516505254 before I merge please?

**Comment by datapythonista** [comment]
Thanks! Do you mind making the change to the title in a new PR, and leaving here only the addition of your name? Sorry if it sounds silly, but in general in open source every PR addresses one and only one thing.

**Comment by beingenfa** [comment]
Sorry, just figuring out things. Now this PR has just addition of my name in it.

**Comment by beingenfa** [comment]
I just realized that I have made changes to my master and put PR from it. I would like to put both changes in separate branches and put separate PR. Closing this PR.

**Comment by beingenfa** [comment]
Added my username in #65 and changed the title in #66. I hope all is good now

**Comment by sara-02** [comment]
@enfageorge kudos. Yes, both the PRs are correctly put :)

**Comment by datapythonista** [comment]
@enfageorge this was closed, and no name was selected for the team. May be you can create an issue to keep that discussion open?

**Comment by beingenfa** [comment]
@datapythonista done in #80 just now

**Pull request closed without merging**